app/common/elasticsearchclient.py

```diff
@@ -2668,20 +2559,30 @@ def build_query_filter(self):
                 self._get_association_rna_range_filter(self.params)
 
     def get_default_size(self):
-        return 25
+        return 300
```

Comment:
this may be onerous.

Reply:
@elipapa @apierleoni and the frontend guys wanted it like that as the difficulty of paginating...